packages/react/src/stories/ic-text-field.stories.mdx

```diff
@@ -482,3 +482,134 @@ export const Uncontrolled = () => {
     ></IcTextField>
   </Story>
 </Canvas>
+
+### Playground
+
+export const defaultArgs = {
+  disabled: false,
+  fullWidth: false,
+  helperText: "",
+  hideLabel: false,
+  inputId: "ic-text-field-input-0",
+  inputmode: "text",
+  label: "Text Field",
+  maxCharacters: 0,
+  maxLength: 0,
+  minCharacters: 0,
+  name: "0",
+  placeholder: "",
+  readonly: false,
+  required: false,
+  resize: false,
+  rows: 1,
+  size: "default",
+  spellcheck: false,
+  type: "text",
+  validationInline: false,
+  validationStatus: "none",
+  validationText: "",
+  debounce: 0
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    argTypes={{
+      disabled: {
+        control: { type: "boolean" }
+      },
+      fullWidth: {
+        control: { type: "boolean" }
+      },
+      hideLabel: {
+        control: { type: "boolean" }
+      },
```

Comment:
Any args with true or false are able to infer the control type without specifying it here, so these should be able to be removed